src/main/kotlin/app/revanced/patches/all/misc/versioncode/ChangeVersionCodePatch.kt

```diff
@@ -11,10 +11,10 @@ import org.w3c.dom.Element
 
 @Patch(
     name = "Change version code",
-    description = "Changes the version code of the app. By default the highest version code is set. " +
-            "This allows older versions of an app to be installed " +
-            "if their version code is set to the same or a higher value and can stop app stores to update the app. " +
-            "This does not apply when installing with root install (mount).",
+    description = "Changes the version code of the app to the value specified in options.json. " +
+            "Except when mounting, this can prevent app stores from updating the app and allow " +
+            "the app to be installed over an existing installation that has a higher version code. " +
```

Comment:
```suggestion
            "the app to be installed over an existing installation. " +
```

Reply:
Apps can be installed over existing installations without the patch. The patch is only needed if the existing application is a higher version

Reply:
Most people won't know what version code is, so instead it should be something simple. Perhaps:
```suggestion
            "lower version of the app to be installed over an existing higher version. " +
```

Reply:
not a big problem
`Change version code` patch is a universal patch and is excluded by default

I assume that only people who know what this patch does will use it